src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3020,10 +3020,17 @@ def create_image_version(cmd, resource_group_name, gallery_name, gallery_image_n
             source = GalleryArtifactVersionSource(id=managed_image)
         if os_snapshot is not None:
             os_disk_image = GalleryOSDiskImage(source=GalleryArtifactVersionSource(id=os_snapshot))
+        if data_snapshot_luns and not data_snapshots:
```

Comment:
just curious, is there general way in CLI to force parameters combination?

Reply:
I don't think we have, otherwise, we don't need so many validation code. This a common code pattern in Azure CLI.